centreon/www/include/Administration/parameters/debug/form.ihtml

```diff
@@ -13,6 +13,8 @@
     </tr>
 	<tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="tip_logs_directory">{$form.debug_path.label}</td><td class="FormRowValue">{$form.debug_path.html}</td></tr>
 	<tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="tip_authentication_debug">{$form.debug_auth.label}</td><td class="FormRowValue">{$form.debug_auth.html}</td></tr>
+	<tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="tip_application_debug">{$form.debug_application.label}</td><td class="FormRowValue">{$form.debug_application.html}</td></tr>
+	<tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="tip_debug_level">{$form.debug_level.label}</td><td class="FormRowValue">{$form.debug_level.html}</td></tr>
```

Comment:
Remove the checkbox and only handle the level listing with a None for debug deactivated